modules/cluster-logging-collector-log-forward-gcp.adoc

```diff
@@ -25,38 +25,44 @@ Using this feature with Fluentd is not supported.
 ----
 $ oc -n openshift-logging create secret generic gcp-secret --from-file google-application-credentials.json=_<your_service_account_key_file.json>_
 ----
+
 . Create a `ClusterLogForwarder` Custom Resource YAML using the template below:
 +
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: <log_forwarder_name> <1>
-  namespace: <log_forwarder_namespace> <2>
+  name: <log_forwarder_name>
+  namespace: openshift-logging
 spec:
-  serviceAccountName: <service_account_name> <3>
+  serviceAccountName: 
+    name: <service_account_name> #<1>
   outputs:
     - name: gcp-1
       type: googleCloudLogging
-      secret:
-        name: gcp-secret
       googleCloudLogging:
-        projectId : "openshift-gce-devel" <4>
-        logId : "app-gcp" <5>
+        id:
+          type : project
+          value: openshift-gce-devel #<2>
+      logId : app-gcp #<3>
+      authentication:
```

Comment:
same here, indentation.    `authentication` was moved under each output type in v6, so this should be a child of `googleCloudLogging`